app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Powerlevel9k theme for ZSH which uses Powerline Fonts."
+HOMEPAGE="https://github.com/bhilburn/powerlevel9k"
+KEYWORDS="~amd64"
+LICENSE="MIT"
+SLOT="0"
+RDEPEND="media-fonts/powerline-symbols"
+SRC_URI="https://github.com/bhilburn/powerlevel9k/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"
+EXTRACT_DIR="powerlevel9k-0.6.6"
+
+src_install()
+{
+	insinto "/usr/share/zsh-theme-powerlevel9k"
+	doins -r {"${EXTRACT_DIR}/debug","${EXTRACT_DIR}/docker","${EXTRACT_DIR}/functions","${EXTRACT_DIR}/shunit2","${EXTRACT_DIR}/test","${EXTRACT_DIR}/test-bsd-vm","${EXTRACT_DIR}/test-vm","${EXTRACT_DIR}/test-vm-providers"}
```

Comment:
this is pretty ugly, if there is no better way of handeling this, you can use the `for` loop